src/main/g8/build.sbt

```diff
@@ -14,22 +15,26 @@ lazy val root = (project in file("."))
     // Options used by `native-image` when building native image.
     // https://www.graalvm.org/docs/reference-manual/native-image/
     graalVMNativeImageOptions ++= Seq(
-        "--initialize-at-build-time", // Auto-packs dependent libs at build-time
         "--no-fallback", // Bakes-in run-time reflection (alternately: --auto-fallback, --force-fallback)
         "--no-server", // Won't be running `graalvm-native-image:packageBin` often, so one less thing to break
-        "--static" // Disable for OSX (non-docker) builds - Forces statically-linked binary, compatible with libc (linux)
+        "--static", // Disable for OSX (non-docker) builds - Forces statically-linked binary, compatible with libc (linux)
+        "--initialize-at-build-time=" + Seq( // Auto-packs dependent libs at build-time
+            "$package$.Main",
+            // "some.class.with.long.initialization.Time"
+        ).mkString(",")
     ),
 
     // Concurrency:
-    libraryDependencies += "io.monix" %% "monix" % "3.2.2",
+    libraryDependencies += "org.typelevel" %% "cats-effect" % "3.2.9",
 
     // Arg Parsing:
-    libraryDependencies += "com.github.scopt" %% "scopt" % "4.0.0-RC2",
+    libraryDependencies += "com.monovore" %% "decline-effect" % "2.2.0",
 
     // Logging:
-    libraryDependencies += "com.typesafe.scala-logging" %% "scala-logging" % "3.9.2",
-    libraryDependencies += "ch.qos.logback" % "logback-classic" % "1.2.3", // See README.md for why we don't use log4j2
+    libraryDependencies += "com.typesafe.scala-logging" %% "scala-logging" % "3.9.4",
+    libraryDependencies += "ch.qos.logback" % "logback-classic" % "1.3.0-alpha10", // See README.md for why we don't use log4j2
```

Comment:
:warning: Sketchy version alert :warning: 

```
[error] Call path from entry point to ch.qos.logback.classic.spi.PackagingDataCalculator.populateFrames(StackTraceElementProxy[]): 
[error] 	at ch.qos.logback.classic.spi.PackagingDataCalculator.populateFrames(PackagingDataCalculator.java:72)
[error] 	at ch.qos.logback.classic.spi.PackagingDataCalculator.calculate(PackagingDataCalculator.java:58)
[error] 	at ch.qos.logback.classic.spi.ThrowableProxy.calculatePackagingData(ThrowableProxy.java:142)
[error] 	at ch.qos.logback.classic.spi.LoggingEvent.<init>(LoggingEvent.java:122)
[error] 	at ch.qos.logback.classic.Logger.buildLoggingEventAndAppend(Logger.java:419)
[error] 	at ch.qos.logback.classic.Logger.filterAndLog_0_Or3Plus(Logger.java:383)
[error] 	at ch.qos.logback.classic.Logger.warn(Logger.java:688)
```

I stepped through the code of alpha 4 or 5 myself because I was scared shitless by this one when I first decided to give it a shot.

In practice, this version isn't as sketchy as it sounds, and I believe that's thanks to how they're rolling out the new sl4j-api that this alpha supports. It's pretty dang rock solid. I've been using these alpha builds of logback classic for almost 2 years now with ZERO issues.